<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>GeoSlab703 MdCn BT</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>520</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>   Rotation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_Rotation">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>	→</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="left_Rotation">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>	←</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Move_down">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Move_right">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>→</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Move_left">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>	←</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Move_up">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>500</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  Scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Scale_increase">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Scale_decrease">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_Circle">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_Square">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_5pointstar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5-point star</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_6pointstar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6-point star</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_8pointstar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8-point star</string>
    </property>
   </widget>
   <widget class="QLabel" name="area">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_area">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Area</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_Triangle">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Triangle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_Hexagon">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hexagon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_perimeter">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Perimeter</string>
    </property>
   </widget>
   <widget class="QLabel" name="perimeter">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_Rectangle">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_1">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_Rhombus">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rhombus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_Ellipse">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ellipse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_coordinates">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>210</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Center of gravity</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_4">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>210</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>255</y>
      <width>49</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_canva">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Canva</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete_canva">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_figure">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>423</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change the figure</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>GeoSlab703 MdCn BT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
